pkg/istio/rbac/common/util.go

```diff
@@ -202,30 +203,73 @@ func MemberToOriginJwtSubject(member interface{}) (string, error) {
 
 // RoleToSpiffe reads athenz role name string, and generates the SPIFFE name of it
 // SPIFFE name format: <athenz domain name>/ra/<role name>
-func RoleToSpiffe(athenzDomainName string, roleName string) (string, error) {
+func RoleToSpiffe(athenzDomainName string, roleName string, enableSpiffeTrustDomain bool) ([]string, error) {
 	if len(athenzDomainName) == 0 {
-		return "", fmt.Errorf("athenzDomainName is empty")
+		return nil, fmt.Errorf("athenzDomainName is empty")
 	}
 	if len(roleName) == 0 {
-		return "", fmt.Errorf("roleName is empty")
+		return nil, fmt.Errorf("roleName is empty")
 	}
-	spiffeName := fmt.Sprintf("%s/ra/%s", athenzDomainName, roleName)
-	return spiffeName, nil
+
+	spiffeNames := []string{
+		fmt.Sprintf("%s/ra/%s", athenzDomainName, roleName),
+	}
+	if enableSpiffeTrustDomain {
+		newSpiffeNames := []string{
```

Comment:
Please set the value to `spiffe://athenz.cloud/ns/{role-athenz-domain}/ra/{athenz-role-name}`